tutorials/tmva/TMVA_CNN_Classification.C

```diff
@@ -224,16 +225,25 @@ void TMVA_CNN_Classification(int nevts = 1000, std::vector<bool> opt = {1, 1, 1,
 
    // TString inputFileName = "tmva_class_example.root";
 
-   auto inputFile = TFile::Open(inputFileName);
+   std::unique_ptr<TFile> inputFile{TFile::Open(inputFileName)};
```

Comment:
I would move the line above, calling MakeImageTree before opening the output TMVA file to avoid having 2 files open at the same time both in write mode.